.github/workflows/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] Summary of item"
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Thanks for reporting this! Please provide as much info as you can about the issue, so we can best help you. As a general guidance, pointers from https://stackoverflow.com/help/mcve can help you craft a good bug report.
+
+If this is a question please ask on [StackOverflow](https://stackoverflow.com/questions/tagged/consul).
+-->
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+
+
+**Steps To Reproduce**
+<!--
+Provide the steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+-->
+
+
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+
+
+**Screenshots** or **Logs**
+<!-- If applicable, add screenshots to help explain your problem. -->
```

Comment:
I think we should emphasize logs over screenshots here, or even remove screenshots altogether.
If user post screenshots of logs/code/config, they won't be very accessible.

Reply:
removed